flytekit/clis/sdk_in_container/serialize.py

```diff
@@ -33,12 +34,24 @@ def serialize_workflows(pkgs):
         tmap[t.id] = t
 
     for m, k, w in iterate_registerable_entities_in_order(pkgs, include_entities={_workflow.SdkWorkflow}):
-        click.echo('Serializing {}'.format(_utils.fqdn(m.__name__, k, entity_type=w.resource_type)))
+        name = _utils.fqdn(m.__name__, k, entity_type=w.resource_type)
+        click.echo('Serializing {}'.format(name))
         task_templates = []
         for n in w.nodes:
             if n.task_node is not None:
                 task_templates.append(tmap[n.task_node.reference_id])
```

Comment:
not this pr, but this isn't correct right?  @matthewphsmith?  there can be duplicates